File: app/src/main/java/com/example/travelc/ViewMoreActivityNamsai.java
package com.example.travelc;

import android.os.Bundle;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class ViewMoreActivityNamsai extends AppCompatActivity {
    private WebView webView;
    private Button backButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_view_more_amarkantak);

        // Find the WebView and Button by their ids
        webView = findViewById(R.id.webView);
        backButton = findViewById(R.id.buttonBack);

        // Enable JavaScript (optional)
        WebSettings webSettings = webView.getSettings();
        webSettings.setJavaScriptEnabled(true);

        // Load HTML file from assets folder
        webView.loadUrl("file:///android_asset/index3.html");

        // Set WebViewClient to handle links within WebView
        webView.setWebViewClient(new WebViewClient());

        // Set OnClickListener for the "Back" button
        backButton.setOnClickListener(view -> {
            // Check if the WebView can go back
            if (webView.canGoBack()) {
                // Go back in WebView's history
                webView.goBack();
            } else {
                // If WebView cannot go back, finish the activity
                finish();
            }
        });
    }
}

File: app/src/main/java/com/example/travelc/ViewMoreActivityNamsai.kt
package com.example.travelc

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.travelc.databinding.ActivityViewMoreNamsaiBinding

class ViewMoreActivityNamsai : AppCompatActivity() {
    private lateinit var binding: ActivityViewMoreNamsaiBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityViewMoreNamsaiBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        // Set OnClickListener for the "Back" option
        binding.buttonBack.setOnClickListener {
            // Finish the current activity to go back to the previous one
            finish()
        }
    }
}
